src/main/webapp/index-script.js

```diff
@@ -390,10 +392,10 @@ function populateReviews(reviewList, placeID) {
   const entireList = document.createElement('ul');
   entireList.id += 'reviews-list';
 
-  if (reviewList == null) {
+  if (reviewList[0] == null) {
```

Comment:
Im not super sure what exactly you are checking for here. If you are just that the array is empty or doesn't exist, i'd feel more comfortable with one of the more standard options here:
https://www.geeksforgeeks.org/check-if-an-array-is-empty-or-not-in-javascript/ .

Reply:
Will do, thanks for the link!